src/main/java/org/mockito/junit/MockitoTestRule.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.junit;
+
+import org.junit.rules.TestRule;
+import org.mockito.Incubating;
+import org.mockito.MockSettings;
+import org.mockito.Mockito;
+import org.mockito.exceptions.misusing.PotentialStubbingProblem;
+import org.mockito.quality.MockitoHint;
+import org.mockito.quality.Strictness;
+
+/**
+ * Equivalent to {@link MockitoRule}, but also inherits from {@link TestRule}. For more information,
+ * please see the documentation on {@link MockitoRule}.
+ */
+public interface MockitoTestRule extends TestRule {
+
+    /**
+     * Rule will not report stubbing warnings during test execution. By default, stubbing warnings are
```

Comment:
I'd write a single sentence here and link to the MockitoRule.silent(). This way it's more DRY but still useful.

Reply:
Done